<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdPowderFitWidget</class>
 <widget class="QDialog" name="QxrdPowderFitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Powder Fit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="m_ImageZoomInButton">
           <property name="toolTip">
            <string>Zoom In</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_ImageZoomOutButton">
           <property name="toolTip">
            <string>Zoom Out</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/zoom-out.png</normaloff>:/images/zoom-out.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_ImageZoomAllButton">
           <property name="toolTip">
            <string>Auto Scale</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/zoom-all.png</normaloff>:/images/zoom-all.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_ImageSetCenterButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/ringfinder.png</normaloff>:/images/ringfinder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_ImageMeasureButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="qxrdresources.qrc">
             <normaloff>:/images/measure.png</normaloff>:/images/measure.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QxrdPowderFitImagePlot" name="m_ImagePlot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>512</width>
           <height>512</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>512</width>
           <height>512</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="m_PowderFitSteps">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="m_IdentifyRingsTab">
          <attribute name="title">
           <string>Identify Rings</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Identify Rings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdNextStepButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="m_IdHelpBrowser">
             <property name="source">
              <url>
               <string>qrc:/help/qxrdpowderfitidentifyhelp.html</string>
              </url>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="m_IdPrevRingButton">
               <property name="toolTip">
                <string>Previous Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="m_IdRingNumber">
               <property name="font">
                <font>
                 <family>Abyssinica SIL</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdNextRingButton">
               <property name="toolTip">
                <string>Next Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdAddRingButton">
               <property name="toolTip">
                <string>Add Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdRemoveRingButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Delete Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdPerformFitButton">
               <property name="toolTip">
                <string>Refine Fit</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdAddPointButton">
               <property name="toolTip">
                <string>Add Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_IdDeletePointButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Delete Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QTableWidget" name="m_IdRingsTable"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="m_RefineFitTab">
          <attribute name="title">
           <string>Refine Fit</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="m_RefPrevStepButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Refine Fit</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="m_RefNextStepButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="m_refHelpBrowser">
             <property name="source">
              <url>
               <string>qrc:/help/qxrdpowderfitrefinehelp.html</string>
              </url>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="m_PerformFitButton">
               <property name="toolTip">
                <string>Refine Fit</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_CancelFitButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Stop Fit</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/process-stop.png</normaloff>:/images/process-stop.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_UndoFitButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Undo Fit</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/edit-undo.png</normaloff>:/images/edit-undo.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="m_AddRingButton">
               <property name="toolTip">
                <string>Add Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_DeleteRingButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Delete Ring</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QTableWidget" name="m_FitParameters">
             <column>
              <property name="text">
               <string>Parameter</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Fit?</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Lim?</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Low Limit</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>High Limit</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="m_ReviewResultsTab">
          <attribute name="title">
           <string>Results</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="m_ResPrevStepButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Review Results</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="m_ResNextStepButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qxrdresources.qrc">
                 <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="m_ResHelpBrowser">
             <property name="source">
              <url>
               <string>qrc:/help/qxrdpowderfitresultshelp.html</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="m_FittingOutput">
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxrdPowderFitImagePlot</class>
   <extends>QWidget</extends>
   <header>qxrdpowderfitimageplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
